integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java

```diff
@@ -41,89 +38,57 @@
  */
 public class TieredStoreIntegrationTest {
   private static final int MEM_CAPACITY_BYTES = 1000;
-  private static final int DISK_CAPACITY_BYTES = 10000;
   private static final int USER_QUOTA_UNIT_BYTES = 100;
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
-  private LocalTachyonCluster mLocalTachyonCluster = null;
-  private TachyonFS mTFS = null;
+  private LocalTachyonCluster mLocalTachyonCluster;
   private TachyonConf mWorkerConf;
+  private TachyonFS mTFS;
 
   @After
   public final void after() throws Exception {
     mLocalTachyonCluster.stop();
-    System.clearProperty(Constants.WORKER_MAX_TIERED_STORAGE_LEVEL);
-    System.clearProperty(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_ALIAS_FORMAT, 1));
-    System.clearProperty(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_PATH_FORMAT, 1));
-    System.clearProperty(
-        String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_QUOTA_FORMAT, 1));
   }
 
   @Before
   public final void before() throws Exception {
     mLocalTachyonCluster =
         new LocalTachyonCluster(MEM_CAPACITY_BYTES, USER_QUOTA_UNIT_BYTES, Constants.GB);
-
-    // Add system properties to pre-populate the storage tiers
-    // TODO Need to change LocalTachyonCluster to pass this info to be set in TachyonConf
-    System.setProperty(Constants.WORKER_MAX_TIERED_STORAGE_LEVEL, "2");
-    System.setProperty(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_ALIAS_FORMAT, 1),
-        "HDD");
-    System.setProperty(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_PATH_FORMAT, 1),
-        "/disk1" + "," + "/disk2");
-    System.setProperty(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_QUOTA_FORMAT, 1),
-        DISK_CAPACITY_BYTES + "");
-
     mLocalTachyonCluster.start();
     mTFS = mLocalTachyonCluster.getClient();
     mWorkerConf = mLocalTachyonCluster.getWorkerTachyonConf();
   }
 
-  // TODO: this test is allocator and evictor specific and really testing LRU.
-  /*@Test
-  public void blockEvict() throws IOException, InterruptedException {
+  @Test
+  public void pinFileTest() throws Exception {
+    // Create a file and then pin it
     int fileId1 =
-        TachyonFSTestUtils.createByteFile(mTFS, "/root/test1", WriteType.TRY_CACHE,
-            MEM_CAPACITY_BYTES / 6);
-    int fileId2 =
-        TachyonFSTestUtils.createByteFile(mTFS, "/root/test2", WriteType.TRY_CACHE,
-            MEM_CAPACITY_BYTES / 6);
+        TachyonFSTestUtils.createByteFile(mTFS, "/test1", WriteType.MUST_CACHE, MEM_CAPACITY_BYTES);
+    mTFS.pinFile(fileId1);
+    CommonUtils.sleepMs(LOG, TestUtils.getToMasterHeartBeatIntervalMs(mWorkerConf) * 3);
+
+    // Try to create a file that cannot be stored unless the previous file is evicted, expect an
```

Comment:
It is better to make this part of exception test should be a separate test case, and use more standard way to expect exception in JUNIT, so we could expect better on exception type and message.